src/Microsoft.SqlTools.Serialization/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.SqlTools.Hosting.Utility;
+using Microsoft.SqlTools.ServiceLayer.SqlContext;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.Serialization
+{
+    /// <summary>
+    /// Main application class for Serialization Service Host executable
+    /// </summary>
+    internal class Program
+    {
+        /// <summary>
+        /// Main entry point into the Serialization Service Host
+        /// </summary>
+        internal static void Main(string[] args)
+        {
+            try
+            {
+                // read command-line arguments
+                CommandOptions commandOptions = new CommandOptions(args, "SqlSerializationService.exe");
```

Comment:
Minor: I'd suggest making the exe name and the log file path as constants for easier readibility later. It's fairly standard in C#